coderedcms/project_template/pro/project_name/settings/base.py

```diff
@@ -194,3 +194,7 @@
 # custom implementation.
 CRX_DISABLE_NAVBAR = True
 CRX_DISABLE_FOOTER = True
+
+# Default to True to maintain existing behavior for existing installations.
+CRX_NO_SITE_NAVBAR_FALLBACK = True
```

Comment:
These settings are not necessary. The project_template is only applied when someone is creating a new site via `coderedcms start`, so backwards compatibility is not necessary. Please remove them to simplify it.